From 3f6bf9c1a77d6d325d596326f604718b2c5dedfb Mon Sep 17 00:00:00 2001
From: Daniel Mach <dmach@redhat.com>
Date: Fri, 19 Jul 2019 07:38:08 +0200
Subject: [PATCH] Define RPMTAG_MODULARITYLABEL if rpm doesn't define it.

This allows us to stay at an older rpm version.
---
 libdnf/dnf-rpmts.cpp | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/libdnf/dnf-rpmts.cpp b/libdnf/dnf-rpmts.cpp
index c9cf1c1fe..e679d36d2 100644
--- a/libdnf/dnf-rpmts.cpp
+++ b/libdnf/dnf-rpmts.cpp
@@ -45,6 +45,11 @@
 #include "utils/bgettext/bgettext-lib.h"
 #include "dnf-package.h"
 
+// older RPM doesn't have RPMTAG_MODULARITYLABEL defined
+#ifndef RPMTAG_MODULARITYLABEL
+#define RPMTAG_MODULARITYLABEL 5096
+#endif
+
 
 static gboolean
 test_fail_safe(Header * hdr, DnfPackage * pkg, GError **error)
